includes/Shopping/WooCommerce_Query.php

```diff
@@ -61,10 +61,14 @@ public function __construct( WooCommerce $woocommerce ) {
 	public function get_search( string $search_term ) {
 		$status = $this->woocommerce->get_plugin_status();
 
-		if ( ! $status['active'] ) {
+		if ( ! $status['installed'] ) {
 			return new WP_Error( 'rest_woocommerce_not_installed', __( 'WooCommerce is not installed.', 'web-stories' ), [ 'status' => 400 ] );
```

Comment:
```suggestion
			return new WP_Error( 'rest_woocommerce_not_installed', __( 'WooCommerce is not installed. Please install and activate it again try again. ', 'web-stories' ), [ 'status' => 400 ] );
```

Reply:
Yeah that should suffice for now.

We can go through all the wording again later if needed. Most important is that all these cases are handled.